.changeset/nervous-hounds-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
This is only a `patch` change whist the backend plugin is below `1.0.0`

Reply:
Done @benjdlambert